src/openeo_grass_gis_driver/actinia_processing/load_collection_process.py

```diff
@@ -291,8 +324,10 @@ def create_process_chain_entry(input_object: DataObject,
                         "param": "s", "value": str(south)}, {
                         "param": "e", "value": str(east)}, {
                         "param": "w", "value": str(west)}, {
-                            "param": "crs", "value": str(crs)}, {
-                                "param": "strds", "value": input_object.grass_name()}, ]}
+                        "param": "crs", "value": str(crs)}, {
+                        "param": "strds", "value": stdr_name},
```

Comment:
Is `stdr_name` also set for type `strds` ?

Reply:
Yeap, I create the name of the `strds` object and use it for the `t.info` and the `r.region`

Reply:
ok then.